app/scripts/modules/oracle/instance/details/instance.details.controller.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+import {INSTANCE_READ_SERVICE} from 'core/instance/instance.read.service';
+import {INSTANCE_WRITE_SERVICE} from 'core/instance/instance.write.service';
+import {CLOUD_PROVIDER_REGISTRY} from 'core/cloudProvider/cloudProvider.registry';
+
+let angular = require('angular');
```

Comment:
just pushed a rather large PR detailing new import structure - can you roll these into:
```javascript
const angular = require('angular');

import { 
  CLOUD_PROVIDER_REGISTRY, 
  INSTANCE_READ_SERVICE, 
  INSTANCE_WRITE_SERVICE 
} from '@spinnaker.core';
```